pom.xml

```diff
@@ -462,6 +463,12 @@
                 <artifactId>spring-context</artifactId>
                 <version>${spring.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.postgresql</groupId>
+                <artifactId>postgresql</artifactId>
+                <version>${postgresql.version}</version>
+                <scope>provided</scope>
```

Comment:
we need to copy this into the libs via dist build. Since this is BSD-2 we should be ok with ASLv2 license